Lesson 1

tehMono opened the pull request.

tehMono committed:
Lesson1

evodnev commented:
По отдельным заданиям замечаний нет.

Не выполнено основное требование ДЗ:
**Домашние задания**
1. Все домашние задания курса выполняем в одном проекте. Храним в GitHub. При выполнении следующего задания не должны удаляться предыдущие. Предусмотреть возможность удобного - на усмотрение автора - вызова нужного примера/задания.

Зачем - основная задача курса - научиться структурировать код. Должен получиться один проект, из которого мы будем выбирать (запускать) нужное задание.

Также рекомендую форматировать код (Edit - Advanced - Format Document), выглядит аккуратно. Когда форматируешь чужой большой файл и получаешь много изменений, приятного мало. Маскируются реальные изменения.

tehMono committed:
Папка для Lesson1

сделал папку для Lesson1

Пока не понимаю, как сделать отдельно вызов для каждого урока!

Спасибо за Форматирование! делал обычно руками, очень было не удобно!

tehMono commented:
Подскажите, как лучше структурировать ! В Git я это сделаю в master и потом буду добавлять туда другие уроки! я правильно понимаю?
